runtime/service.go

```diff
@@ -738,3 +754,30 @@ func packBundle(path string, options *ptypes.Any) (*ptypes.Any, error) {
 	}
 	return ptypes.MarshalAny(extraData)
 }
+
+// setupStubDrive will create a flat file with the drive ID as its contents.
+// The file is padded with null bytes to a minimum size of minFCDriveImageSize.
+// This is due to Firecracker minimum drive image size limits.
+func setupStubDrive(builder firecracker.DrivesBuilder, dir, driveID string) (firecracker.DrivesBuilder, error) {
+	path := filepath.Join(dir, driveID)
+	f, err := os.Create(path)
+	if err != nil {
+		return builder, err
+	}
+	defer f.Close()
+
+	if _, err = f.WriteString(driveID); err != nil {
+		return builder, err
+	}
+
+	// Firecracker will only show drives that are multiples of 512.
```

Comment:
Multiples or minimum?

Right now there's no bound on the size of the `driveID` string, so we could easily have a string that exceeds 512 bytes.

Reply:
if the disk image is not a multiple of `SECTOR_SIZE` which in this case is 512, it will not show the remaining bytes. So the minimum needs to be 512. I'll reword as the current comment doesn't really say that.